### PR TITLE
Test "any account can return an escrow"

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: packages/iov-bns/src/bnsconnection.spec.ts [hunk 1]
```diff
@@ -1104,6 +1111,63 @@ describe("BnsConnection", () => {
       connection.disconnect();
     });
 
+    it("any account can return an escrow", async () => {
```
Comment (willclarktech) [inline packages/iov-bns/src/bnsconnection.spec.ts]: This description makes it sound like anyone can cancel anyone else's escrow.

File: packages/iov-bns/src/bnsconnection.spec.ts [hunk 2]
```diff
@@ -1104,6 +1111,63 @@ describe("BnsConnection", () => {
...
+
+        const timeout = Math.floor(Date.now() / 1000) + 3;
+
+        const createEscrowTx = await connection.withDefaultFee<CreateEscrowTx & WithCreator>({
+          kind: "bns/create_escrow",
+          creator: sender,
+          sender: senderAddress,
+          arbiter: encodeBnsAddress("tiov", fromHex("0000000000000000000000000000000000000000")),
+          recipient: encodeBnsAddress("tiov", fromHex("0000000000000000000000000000000000000000")),
+          amounts: [defaultAmount], // no tokens
```
[the remark this answers is not in the document]
Comment (webmaster128) [reply packages/iov-bns/src/bnsconnection.spec.ts]: Thank! I tried and empty array first but the chain rejected it

File: packages/iov-bns/src/bnsconnection.spec.ts (hunk 1 above)
Comment (webmaster128) [reply packages/iov-bns/src/bnsconnection.spec.ts]: This is exactly what happens, but only after timeout. Any idea for better wording? I wanted to pick up the verb from the transaction name

Comment (willclarktech) [reply packages/iov-bns/src/bnsconnection.spec.ts]: Ah yeah, I remember now. Perhaps just "any account can return an escrow (after the timeout)".

Comment (webmaster128) [reply packages/iov-bns/src/bnsconnection.spec.ts]: This behaviour is of critical importance when we want to use escrows for vesting, which after the lockup period send to a Ledger address. This Ledger address can only sign send transactions, so "a friend" is needed to help out triggering the returns.

**Pull request merged**